setup.py

```diff
@@ -26,7 +26,8 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["urllib3 >= 1.15", "six >= 1.10", "certifi", "python-dateutil", "requests"]
+with open("requirements.txt") as requirements:
+    REQUIRES = [req.replace('\n', '') for req in requirements.readlines()]
```

Comment:
There should be double quotes here in accordance with PEP8. Just run this file through black auto-formatter to make it compliant.
```suggestion
    REQUIRES = [req.replace("\n", "") for req in requirements.readlines()]
```